Update hvnc.md

jtdendunnen opened the pull request:
updated to shwo terms end mid-year

jtdendunnen committed:
Update hvnc.md

updated to shwo terms end mid-year

ifokkema commented on docs/hvnc.md:
```diff
@@ -22,15 +22,15 @@ The HVNC started in 2013, initiated by Global Variome (GV) / the Human Variome P
 | Matthew Stachowiak           | GenomOncology, United States                                        | 2024 - 2028 |
-| Rachel Taylor                | University of Manchester, United Kingdom                            | 2022 - 2025 |
-| Alex Wagner                  | Nationwide Childrens Hospital, US                                   | 2022 - 2025 |
+| Rachel Taylor                | University of Manchester, United Kingdom                            | 2022 - 2026 |
```
Rachel left last year, so she should actually be moved to the past members.

ifokkema reviewed (changes requested):
Oops, I see I overlooked this. Rachel left the committee last year, so she should be moved to the past members.

ifokkema committed:
Several updates to the HVNC member list.
- Marked Johan and Alex as co-chairs;
- Removed Rachel Taylor as she stepped down;
- Formatted the table, making it easier to read in Markdown.

ifokkema committed:
Laura Conlin replaced Ros Hastings as the ISCN representative.

ifokkema committed:
Better left-align the terms column, while we're at it.

ifokkema committed:
Added the terms for the co-chairs.

Since Alex will be co-chair until 2028, his HVNC membership
 automatically renews to 2030. Since Johan's co-chair term ends in
 2030, his HVNC membership automatically renews to 2030.
